project/Build.scala

```diff
@@ -119,6 +129,38 @@ object MyScalaJSPlugin extends AutoPlugin {
         (fullClasspath in (Build.linker.v2_12, Runtime)).value
       },
 
+      scalaJSLoggerFactory := {
+        val default = scalaJSLoggerFactory.value
+
+        ExposedValues.autoImport.loggerTimingOut.?.value match {
+          case None => default
+
+          case Some(file) =>
+            /* Instrument logger to dump calls to `time` to a file.
+             * The way we manage the file is a hack: It is difficult to properly
+             * manage resources in sbt's settings system, so we simply set the
+             * printStream to autoFlush, avoiding that we have to close the file.
+             */
+            val stream = new PrintStream(new FileOutputStream(file, /*append=*/true), /*autoFlush=*/true)
```

Comment:
Perhaps specify an explicit `StandardCharsets.UTF_8` here as well. On Windows it's typically not the default (at least before JDK 20ish).

Reply:
Good point. Needed to specify a string, since `PrintStream` doesn't accept a Charset (at least on JDK 8).